### ISSUE TITLE
Gltf attribute attenuation_distance does not have a default value

### ISSUE THREAD

**Issue by JGamache-autodesk** [AcademySoftwareFoundation/MaterialX#1136]
As [seen here](https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/bxdf/gltf_pbr.mtlx#L27):
```xml
    <input name="attenuation_distance" uniform="true" type="float" uimin="0" uiname="Attenuation Distance" uifolder="Volume" />
```
And if I read the [specs](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_volume/README.md#attenuation) correctly, the default should be `+infinity`. Should we represent that using FLOAT_MAX?

**Comment by JGamache-autodesk** [comment]
It is going to be interesting to build a UI to manipulate that value.

**Comment by jstone-lucasfilm** [comment]
Good point, @JGamache-autodesk, and I wonder if we should select a suitably-large finite value as the canonical default, for the benefit of UI systems that don't support infinity.
